<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SeatWidget</class>
 <widget class="QWidget" name="SeatWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>152</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>GridInScreenWidget</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="acceptDrops">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QWidget" name="headWidget" native="true">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>20</y>
        <width>61</width>
        <height>61</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="acceptDrops">
       <bool>true</bool>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="headLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QFrame" name="vCheckFrame">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>20</y>
        <width>20</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/image/stopvideo.png);</string>
      </property>
     </widget>
     <widget class="QLabel" name="nameLabel">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>96</y>
        <width>40</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Agency FB</family>
        <pointsize>6</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QFrame" name="aCheckFrame">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>20</y>
        <width>20</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/image/stopaudio.png);</string>
      </property>
     </widget>
     <widget class="QLabel" name="documentLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>62</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>屏幕共享</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
  <action name="highQualityVideoAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>高清</string>
   </property>
   <property name="toolTip">
    <string>接收高清视频</string>
   </property>
  </action>
  <action name="lowQualityVideoAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>低清</string>
   </property>
   <property name="toolTip">
    <string>接收低清视频</string>
   </property>
  </action>
  <action name="stopReceiveAction">
   <property name="text">
    <string>停止接收</string>
   </property>
   <property name="toolTip">
    <string>停止接收</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
